### ISSUE TITLE
Environment Section Context

### ISSUE THREAD

**Issue by lmbrainerd** [acortell3/archaeoriddle#39]
https://github.com/acortell3/archaeoriddle/blob/344961c687b0c9b7792d30f09268aa089c9e5aa5/doc/bookdown/02_Generate_environment.Rmd#L20

So I have no clue what is happening for a large part of the environment section. There is not enough information to know what the function are doing or what I am supposed to be doing running the code. It would be helpful the top to have a rundown of what we are going to do (Generate, land, make elevation, make settlements). 

In the line highlights, I don't know what the environment function is. If @simoncarrignon or @acortell3 could give me an informal rundown on what each step is I can try making some overviews on my breaks.

**Comment by simoncarrignon** [comment]
yup sorry ; your tapping into my stuff :// I still need to go through all that and clean (wait until you reach sections 06_ and 07_ :D )
